.github/workflows/ci.yml

```diff
@@ -29,6 +29,11 @@ jobs:
         with:
           go-version: 1.18
 
-      - uses: golangci/golangci-lint-action@v3
-        with:
-          version: v1.46
+      - run: |
+          if [ ! -z $(gofmt -l .) ]; then echo 'Make sure to run "go fmt ./..." before commit!' && exit 1; fi
+          go test ./... -race -v -count=1 -cover
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_PAT }}
+          DOCKER_HOST: tcp://127.0.0.1:2375
+          DOCKER_API_VERSION: "1.41"
+          SUPABASE_API_URI: http://127.0.0.1:2376
```

Comment:
Any reason to change the order of these GHA jobs? I would prefer minimising changes in this PR.

Reply:
Yes, lint is common to run before tests, because it is faster and will fail pipeline faster

Reply:
Jobs on GitHub actions are run in parallel by default https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow

Reply:
Oh, damn, you right, I have not noticed that these are jobs and not steps
Will fix it, thanks 👍